website/docs/api/candidate.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Candidate
```

Comment:
Looks like we've now got this information duplicated in the separate file `candidate.md` as well as in `kb.md`

Reply:
We just want to remove this one, right?